_pages/endpoint.html

```diff
@@ -6,6 +6,11 @@
     <form>
         <input id="endpoint_txt-url" type="text" placeholder="Custom endpoint url" autocomplete="on"/>
         <input id="endpoint_txt-appid" type="text" placeholder="App ID of endpoint" autocomplete="on"/>
+        Brand:
```

Comment:
![image](https://user-images.githubusercontent.com/38449124/71611823-147e3580-2bd7-11ea-83fc-30d83f174d19.png)
Could we add labels for endpoint and appid? It looks a bit odd now that 1/3 fields have a label.

Reply:
@billmarriott-binary Done, please check again. Thanks.

Reply:
Nice, thanks!